studiopress/atomic-blocks
# Remove default font value in favor of theme font size

By removing the default value, users can clear the font size output and use their theme font sizes instead.

**Summary of change:**
- Set default value on attribute to null
- Added check for font value before outputting class

**This PR has been:**

**Have the changes in this PR been added to the documentation for this project?**
Does not apply

**How to test:**
1. Check out the 2.0 branch
2. Add an accordion block to the page, set a font size, save the page
3. Switch to issue/171 branch
4. Refresh the page, clear the default font value from the slider, and save the page
5. Check to make sure the block doesn't break and that a font class wasn't output on the front end

**Fixes:** #171
